components/ManualPage/ManualPage.module.css

```diff
@@ -16,64 +20,51 @@
 }
 
 .pageName {
-    font-size: 2.5rem;
+    font-size: 128px;
 }
 
-.templateImage {
-    width: 100%;
-    position: relative;
-    border-radius: 4px;
+.Manual__image {
+    border-radius: 16px;
     box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
+    object-fit: contain;
+    width: 100% !important;
+    position: relative !important;
```

Comment:
Let's remove important. Why are they needed?

Reply:
I looked at this [article](https://dev.to/felixhaeberle/responsive-fix-for-the-next-js-image-component-1351) to get adaptive pictures with `next/image` (without important it doesn't work)